index.html

```diff
@@ -160,54 +202,52 @@ <h3 class="small-txt">LET'S CONNECT</h3>
     </div>
        <div class="project-check"> <button type="button">Get my resume</button> </div>
 
-  </section>
-
-  <section class="programs-section">
-
-      <ul>
-        <li class="last-labels"> 
-          <h3>Languages</h3> <img src="Images/Arrow_down.svg" alt="Arrow_down">
-        </li>
-
-        <li class="languages-icons">
-          <img src="Images/Javascript-Icon.png" alt="Icon jss"> <span class="lng-txt">JavaScript</span>
-        </li> 
-        <li class="languages-icons">
-          <img src="Images/HTML-5-Icon.png" alt="Icon html">  <span class="lng-txt">HTML</span> 
-        </li>
-        <li class="languages-icons">
-          <img src="Images/CSS-Icon.png" alt="Icon css">  <span class="lng-txt">CSS</span>
-        </li>
-      </ul>
+   </div>
 
-      <ul>
-        <li class="last-labels">
-           <h3>Frameworks</h3> <img src="Images/Arrow_right.svg" alt="Arrow_right">
+   <ul class="language-shapes">
+    <li class="last-labels">
+      <header class="list-header">
+        <h4 class="list-title">Languages</h4>
+        <img src="Images/Arrow_down.svg" alt="Arrow_down">
+      </header>
+      <div class="list-item-content">
+        <ul class="language-list">
+          <li id="js-shape" class="languages-icons">
+            <img src="Images/Javascript-Icon.png" alt="Icon jss" class="language-img">
+            <p class="language-label">JavaScript</p>
+          </li>
+          <li id="html-shape" class="languages-icons">
+            <img src="Images/HTML-5-Icon.png" alt="Icon html" class="language-img">
+            <p class="language-label">HTML</p>
+          </li>
+          <li id="css-shape" class="languages-icons">
+            <img src="Images/CSS-Icon.png" alt="Icon css" class="language-img">
+            <p class="language-label">Css</p>
           </li>
         </ul>
-        
-        <img src="Images/Line-Separator.png" alt="Separator">
-        
-      <ul>
-        <li class="last-labels"> 
-          <h3>Skills</h3> <img src="Images/Arrow_right.svg" alt="Arrow_right">
-        </li>
-      </ul> 
-      
-      <img src="Images/Line-Separator.png" alt="Separator">
-      
+      </div>
+    </li>
+    <li class="last-labels border-bottom">
+      <header class="list-header">
+        <h4 class="list-title">Frameworks</h4>
+        <img src="Images/Arrow_right.svg" alt="Arrow_right">
+      </header>
+    </li>
+    <li class="last-labels border-bottom">
+      <header class="list-header">
+        <h4 class="list-title">Skills</h4>
+        <img src="Images/Arrow_right.svg" alt="Arrow_right">
+      </header>
+    </li>
+  </ul>
+          
   </section>
 
   <section class="all-contact">
```

Comment:
Kindly fill the grey background with white color at the top left corner of the contact me section as shown below 🤗

Your design:
![image](https://user-images.githubusercontent.com/73714615/186942147-4f392d02-99ea-4d64-a77c-05cdc32d159b.png)

Figma design:
![image](https://user-images.githubusercontent.com/73714615/186942305-f719dea9-7002-45a8-87e6-ef44178e558d.png)